prod.yml

```diff
@@ -2,7 +2,17 @@
 # All Rights Reserved.
 
 ---
-- hosts: socorro
+
+- hosts: vagrant
```

Comment:
I don't know what exactly is happenning here but according to the code it looks like something quite orthogonal to socorro installation. Why it's not a separate role?